go/private/common.bzl

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GO_TOOLCHAIN = "@io_bazel_rules_go//go:toolchain"
+GO_TOOLCHAIN = Label("@io_bazel_rules_go//go:toolchain")
```

Comment:
How about applying `str` here instead?

Reply:
I also thought of adding one more constant as a string, but let's go with your suggestion. Thanks! :)